src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -45,7 +45,8 @@ public class AudioController implements AudioApi {
     @Override
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(contextId = HEARING_ID,
-        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, LANGUAGE_SHOP_USER, RCJ_APPEALS})
+        securityRoles = {JUDGE, REQUESTER, APPROVER, TRANSCRIBER, LANGUAGE_SHOP_USER, RCJ_APPEALS},
```

Comment:
Wondering why we would have JUDGE also under securityRoles, if we're thinking that they will never actually have a link to courthouses via `security_group_courthouse_ae`?

Reply:
Its for failover in case a judge is not in a global security group.